app/models/journeyDomain/incident/IncidentDomain.scala

```diff
@@ -31,7 +31,7 @@ case class IncidentDomain(
   endorsement: Option[EndorsementDomain]
 ) extends JourneyDomainModel {
 
-  override def routeIfCompleted(userAnswers: UserAnswers): Option[Call] =
+  override def routeIfCompleted(userAnswers: UserAnswers, stage: Stage): Option[Call] =
     Some(???) // TODO link to next journey
```

Comment:
Guessing this is intentional at this stage?

Reply:
Yep, this will hook into the next journey as there isnt a main CYA. The 'stage' part is brought in from departures to help handle loops that only have one question, it may be it is not needed in arrivals and can be refactored out laters :smile: